style.css

```diff
@@ -0,0 +1,205 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+.header {
+  position: absolute;
+  width: 375px;
+  height: 44px;
+  left: calc(50% - 375px / 2);
+  top: 0;
+}
+
+.logo-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: absolute;
+  width: 1.25rem;
+  height: 40px;
+  left: 12px;
+  top: calc(50% - 40px / 2);
+}
+
+.logo-container a {
+  width: 100px;
+  height: 20px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 1.25rem;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  text-decoration: none;
+}
+
+.nav-menu {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+}
+
+.hamburger {
+  background: transparent;
+  border: none;
+  width: 1.25rem;
+  height: 1.25rem;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.union {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: space-between;
+  height: 1.25rem;
+  width: 1.25rem;
+  cursor: pointer;
+  background: transparent;
+}
+
+.vector1 {
+  position: absolute;
+  left: 16.67%;
+  right: 16.67%;
+  top: 70.83%;
+  bottom: 29.17%;
+  border: 2px solid #6070ff;
+  border-radius: 5px;
+  background: #6070ff;
+}
+
+.vector2 {
+  position: absolute;
+  left: 16.67%;
+  right: 16.67%;
+  top: 50%;
+  bottom: 50%;
+  border: 2px solid #6070ff;
+  border-radius: 5px;
+  background: #6070ff;
+}
+
+.vector3 {
+  position: absolute;
+  left: 16.67%;
+  right: 16.67%;
+  top: 29.17%;
+  bottom: 70.83%;
+  border: 2px solid #6070ff;
+  border-radius: 5px;
+  background: #6070ff;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+  background: url('images/header-shapes mobile@2x.png') no-repeat, url('images/header-background-mobile@2x.png') no-repeat;
+  background-size: 100% 100%;
+}
+
+h1 {
+  width: 327px;
+  height: 150px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+p {
+  margin-top: 20px;
+  width: 327px;
+  height: 146px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
```

Comment:
- [ ] many CSS lines here are repeating and unnecessary. for example here `flex: none` `order: 1` and `flex-grow: 0` is unnecessary because you have not taken `display: flex` so there is no use for that. and I can see this pattern repeating a lot which is against the DRY rules of CSS. kindly consider avoiding using unnecessary CSS code.
- [ ] The color of this `p` tag is not similar to the Figma design. kindly consider using the same color for this element as in the Figma design.

Reply:
p color on figma seems the same color, if i am not mistaken [https://photos.app.goo.gl/SgXBDFQKRLjWV9HN8](url)